config/config.go

```diff
@@ -815,6 +842,33 @@ func (c *Config) parseHistogramBuckets() ([]float64, error) {
 	return result, nil
 }
 
+func (c *Config) parseForwardedHeaders() error {
+	for _, header := range c.ForwardedHeadersList.values {
+		switch header {
+		case "X-Forwarded-For":
+			c.ForwardedHeaders.For = true
+		case "X-Forwarded-For-prepend":
+			c.ForwardedHeaders.PrependFor = true
+		case "X-Forwarded-Host":
+			c.ForwardedHeaders.Host = true
+		case "X-Forwarded-Proto=http":
+			c.ForwardedHeaders.Proto = "http"
+		case "X-Forwarded-Proto=https":
+			c.ForwardedHeaders.Proto = "https"
```

Comment:
Should we also handle https://datatracker.ietf.org/doc/html/rfc7239#section-4 (more short https://en.wikipedia.org/wiki/X-Forwarded-For#Alternatives_and_variations)?
Since X-Forwarded-For was AFAIK never a standard, there is now RFC7239 that should change.
If not we might want to create an issue for this.

Reply:
Yes, I decided to keep `Forwarded` header out of scope to keep PR smaller, it could be added later on - I intentionally did not put `x-` prefix into the flag names such that we could add `Forwarded` or `Via` in the future